tests/apps/courses/test_cms_plugins_organization.py

```diff
@@ -50,18 +50,15 @@ def test_cms_plugins_organization_render(self):
             fill_banner=True,
             fill_logo=True,
             fill_description=True,
+            should_publish=True,
```

Comment:
then should we check the result when the organization is not published as well? on a draft page? on a published page? to cover all the case sof your template.

Reply:
Good point... I'll see what i can do today.

Reply:
It's done but i need to use 3 assetContains to check the complete presence of this tag:
<a class="organization-plugin__body" href="{{ instance.organization.public_extension.extended_object.get_absolute_url }}"
          title="{{ instance.organization.public_extension.extended_object.get_title }}"
          alt="{{ instance.organization.public_extension.extended_object.get_title }} {% trans 'logo' %}">

This doesn't seem good... Is it because it's on multiple lines?